app/controllers/transcriptions_controller.rb

```diff
@@ -41,6 +50,17 @@ def update
     render jsonapi: @transcription
   end
 
+  def unlock
+    @transcription = Transcription.find(params[:id])
+    authorize @transcription, :update?
+    if @transcription.locked_by != current_user.login
+      raise ActionForbiddenError,
```

Comment:
it might be better to name the error `LockedByAnotherUser` or something vs the `ActionForbiddenError` forbidden error, when i read that raise / rescue i'll know the specific reason it's raising, currently i have to read the text below to understand why it's forbidden.

Reply:
I like `LockedByAnotherUser` or `LockedByAnotherUserError`- I feel like every error I see is named with the word 'error' at the end, any thoughts about sticking to this convention? Does it matter?